src/test/kotlin/com/emberjs/translations/EmberI18nIndexTest.kt

```diff
@@ -43,6 +44,7 @@ class EmberI18nIndexTest : BasePlatformTestCase() {
         FileBasedIndex.getInstance().requestRebuild(EmberI18nIndex.NAME)
 
         PlatformTestUtil.dispatchAllEventsInIdeEventQueue()
+        IndexingTestUtil.waitUntilIndexesAreReady(project)
```

Comment:
Tests were failing with this error:
```
com.intellij.ide.startup.ServiceNotReadyException: index ember.i18n has status REQUIRES_REBUILD
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:805)
	at com.intellij.util.indexing.FileBasedIndexEx.processExceptions(FileBasedIndexEx.java:296)
	at com.intellij.util.indexing.FileBasedIndexEx.processValueIterator(FileBasedIndexEx.java:391)
	at com.intellij.util.indexing.FileBasedIndexEx.processValuesInScope(FileBasedIndexEx.java:357)
	at com.intellij.util.indexing.FileBasedIndexImpl.processValuesInScope(FileBasedIndexImpl.java:1113)
	at com.intellij.util.indexing.FileBasedIndexEx.processValues(FileBasedIndexEx.java:276)
	at com.intellij.util.indexing.FileBasedIndexEx.processValues(FileBasedIndexEx.java:264)
	at com.emberjs.translations.EmberI18nIndex$Companion.getTranslations(EmberI18nIndex.kt:64)
	at com.emberjs.translations.EmberI18nIndexTest.doTest(EmberI18nIndexTest.kt:51)
	at com.emberjs.translations.EmberI18nIndexTest.doTest$default(EmberI18nIndexTest.kt:48)
	at com.emberjs.translations.EmberI18nIndexTest.testSimpleKey(EmberI18nIndexTest.kt:12)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit.framework.TestCase.runTest(TestCase.java:177)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$11(UsefulTestCase.java:483)
	at com.intellij.testFramework.UsefulTestCase.lambda$wrapTestRunnable$13(UsefulTestCase.java:504)
	at com.intellij.testFramework.UsefulTestCase.runTestRunnable(UsefulTestCase.java:408)
```

Probably related to this change in the [release notes](https://blog.jetbrains.com/idea/2024/04/intellij-idea-2024-1/)
>  Basic IDE functionalities like code highlighting and completion now work for Java and Kotlin during project indexing, which should enhance your startup experience.